services/core/OpenADRVenAgent/requirements.txt

```diff
@@ -0,0 +1,4 @@
+isodate==0.6
+signxml==2.4
+SQLAlchemy==1.1.4
```

Comment:
Is there any reason why you haven't updated to 2.0 of SQLAlchemy?

Reply:
@craig8 Do you mean SQLAlchemy version 1.2.0, released 12/27/2017?